src/main.zig

```diff
@@ -2710,6 +2711,20 @@ fn buildOutputType(
         try rpath_list.appendSlice(paths.rpaths.items);
     }
 
+    if (builtin.target.os.tag == .windows and
+        target_info.target.os.tag == .windows and
+        target_info.target.abi == .msvc and
+        external_system_libs.len != 0)
+    {
+        var sdk = try windows_sdk.ZigWindowsSDK.find(gpa);
+        defer sdk.free(gpa);
+
+        var libc_installation = LibCInstallation{};
+        try libc_installation.findNativeMsvcLibDir(.{ .allocator = arena, .target = cross_target.toTarget() }, &sdk);
+        try libc_installation.findNativeKernel32LibDir(.{ .allocator = arena, .target = cross_target.toTarget() }, &sdk);
+        try lib_dirs.appendSlice(&.{ libc_installation.msvc_lib_dir.?, libc_installation.kernel32_lib_dir.? });
+    }
+
```

Comment:
We should be able to replicate macOS SDK logic of handling this in `detectLibCIncludeDirs`. I see though that `.msvc` is not treated as native ABI by us, so perhaps this info should be provided explicitly by the user? Or perhaps Windows should have two native ABIs, `.msvc` and `.gnu`?

Reply:
This logic runs before the logic in Compilation that calls `detectLibCIncludeDirs`, are you suggesting to also call that logic here as well?

The native ABI part is a bit tricky as well. For example, if you manually specify the target, I think it's not treated as the native ABI even if they are the same.